cloud/services/internalloadbalancers/internalloadbalancers.go

```diff
@@ -89,9 +84,8 @@ func (s *Service) Reconcile(ctx context.Context, spec interface{}) error {
 					{
 						Name: &frontEndIPConfigName,
 						FrontendIPConfigurationPropertiesFormat: &network.FrontendIPConfigurationPropertiesFormat{
-							PrivateIPAllocationMethod: network.Static,
+							PrivateIPAllocationMethod: network.Dynamic,
```

Comment:
This is easier for now but we could also calculate a static IP based on the vnet address space

Reply:
Is this for the internal load balancer? If so, doesn't it need to be static in case kubelet might use the IP as the api server endpoint?

Reply:
yeah I think you're right... I couldn't think of an obvious way to get a static IP in the vnet when the CIDR is different from the default CIDR but I think that's a blocker for merging this PR.